backend/coins/coin/codes.go

```diff
@@ -40,6 +40,13 @@ const (
 	// There are some more coin codes for the supported erc20 tokens in erc20.go.
 )
 
+const (
```

Comment:
why did you put these in the generic `coin` package instead of using them locally in the `btc` package?

Reply:
With the last update it is also used by `coin` pkg, I left it there, since including `btc` package generates a inclusion cycle.